tests/console/quota.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: test that uses quota command line tool to test
+# This test consists on quota, configuring and use a regular user to test quota .
+# If succeed, the test passes, proving that the connection is working.
+#
+# Maintainer: Marcelo Martins <mmartins@suse.cz>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub setup {
+    my $self = shift;
+    my $username = $testapi::username;
+    my $passwd   = $testapi::password;
+
+    # create filesystem image to use
+    assert_script_run "dd if=/dev/zero of=/tmp/quota.img bs=10M count=10";
+    assert_script_run "mkfs.ext3 -m0 /tmp/quota.img";
+    assert_script_run "mkdir /tmp/quota"; 
+
+    #mount disk image
+    assert_script_run "mount -o loop,rw,usrquota,grpquota /tmp/quota.img /tmp/quota/";
+
+    #creating some dir
+    assert_script_run "cd /tmp/quota/ ; mkdir test-directory ; chmod 77 test-directory ; ls -l";
+}
+
+sub run {
+    my $self = shift;
+    my $username = $testapi::username;
+    my $passwd   = $testapi::password;
+
+    select_console 'root-console';
+
+    # install requirements
+    zypper_call 'in quota quota-nfs';
+
+    # restart quota service
+    systemctl "restart quotaon";
+
+    # setup config files
+    setup;
+    #testing quota commands:
+    assert_script_run "quotacheck -cug /tmp/quota";
+    #setquota to user testuser 
```

Comment:
I guess user is not a testuser

Reply:
Correct, trash from older tests on comment. changing comment.